packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -52,8 +52,7 @@ You can also [configure a custom email address](#about-custom-email-addresses) t
 
 * If you use an Okta-managed TLS certificate, you don't need a [Certificate Authorization Authority (CAA)](https://datatracker.ietf.org/doc/html/rfc6844) record. However, if you do have a CAA record, keep the following in mind:
 
-    * If it's your first time setting up a custom domain URL with an Okta-managed certificate, you need to add `letsencrypt.org` to the issuers list or Okta can't get the TLS certificate. See [Let's Encrypt - Using CAA](https://letsencrypt.org/docs/caa/).
-    * If you already have an Okta-managed TLS certificate and you later get a CAA record, Okta may be unable to renew your certificate.
+    * If you already have an Okta-managed TLS certificate and you later get a CAA record, Okta will be unable to renew your certificate unless you add letsencrypt.org to the issuers list, or remove the CAA record. See [Let's Encrypt - Using CAA](https://letsencrypt.org/docs/caa/).
```

Comment:
I think it would be good to keep the first item.  It is still true.  These are the two ways customers can get into trouble with CAA records.  The first one will prevent them from even finishing the configuration.  The second one will prevent Okta from renewing and is a silent failure.

Reply:
thanks Dan - brought back the first bullet for both new and existing use cases